lib/tokenizers.js

```diff
@@ -235,12 +235,13 @@ async function _createTokenizer() {
   }
 
   // 4. Create a keystore and HMAC key for the tokenizer.
-  const keystore = await kms.createKeystore(
-    {capabilityAgent, referenceId: 'primary'});
+  const keystore = await kms.createKeystore({
+    controller: capabilityAgent.id, referenceId: 'primary'
+  });
   const keystoreAgent = kms.getKeystoreAgent(
-    {capabilityAgent, keystore});
+    {capabilityAgent, keystoreId: keystore.id});
   const key = await keystoreAgent.generateKey(
-    {type: 'hmac', kmsModule: bedrock.config.tokenizer.kmsModule});
+    {type: 'hmac', kmsModule: bedrock.config.tokenizer.kms.defaultKmsModule});
```

Comment:
`kmsModule` is no longer an option for `generateKey`, so this can be improved to a single line that just passes `{type: 'hmac'}` (github wouldn't let me suggest this).

Reply:
Got it.